CONTRIBUTING.md

```diff
@@ -28,35 +19,33 @@ After making these replacements, this file will be ready for use in your project
 3. **Clone Your Fork Locally**:
    - Once your fork is created, you can clone it to your local machine to start working:
    ```bash
-   git clone https://github.com/your-username/[PROJECT_NAME].git
-   cd [PROJECT_NAME]
+   git clone https://github.com/your-username/package_name.git
+   cd package_name
```

Comment:
Same here this is the repo name, not the package name, however, it might be easier to just leave this one as something like: ``cd <repo dir>``. @ryanhausen is there something in ``project_setup.py`` that will pick up just the repo dir name?

Reply:
Yes, use repo dir enhance flexibility!

Reply:
Cool, thanks! But for clarity, it's not a point of flexibility, but that you used the repo URL and repo names don't have to match project/package names. So if/when they differ, this would break.